test/unit/call_test.lua

```diff
@@ -141,7 +141,10 @@ g.test_timeout = function()
 
     t.assert_equals(results, nil)
     t.assert_str_contains(err.err, "Failed for %w+%-0000%-0000%-0000%-000000000000", true)
-    t.assert_str_contains(err.err, "Timeout exceeded")
+    -- 'Timeout exceeded' or 'timed out'.
+    --
+    -- See https://github.com/tarantool/tarantool/pull/6538.
+    t.assert_str_contains(err.err, "[Tt]imed? ?out", true)
```

Comment:
Alternatively you can implement special helper for that.

To be consistent with vshard (https://github.com/tarantool/vshard/commit/f4e57ba2285ec4dfbebaccfc882390beb6c77764#diff-d7277fd5f081018a7522656bb0d6fa5b34b670ed5100ffc99e18373720e6e570R206) and cartridge (https://github.com/tarantool/cartridge/commit/73686c1cfd6a71c776ff2ebac125d295a9a3cde4#diff-aed3b7f006a0175c125574a5621fba5978870e2247cb6d719836f1386718062bR203-R207)

Reply:
It looks more accurate in the code, but will not show the actual error message value in case of an error. Can we achieve both, what do you think?

Reply:
You can move this assert to separate helper. It will show error message and reduce amount of copy-paste in case if there will be more tests with timeout error check.

Also feel free to ignore this message, it's not so serious issue and current patch is also OK for me :)

Reply:
I wrote a simple helper.